dockstore-integration-testing/src/test/java/io/dockstore/client/cli/GeneralIT.java

```diff
@@ -197,7 +155,7 @@ private String getPathfromDB(String type) {
     public void testListAvailableContainers() {
 
         final long count = testingPostgres.runSelectStatement("select count(*) from tool where ispublished='f'", long.class);
-        assertEquals("there should be 4 entries, there are " + count, 4, count);
+        assertEquals("there should be 4 entries, there are " + count, 6, count);
```

Comment:
Mismatch count in the error message. You can leave the number out of our error message. It will automatically say expected 6 actual w/e